next-seo.config.js

```diff
@@ -8,10 +8,10 @@ export default {
     site_name: 'cert-manager',
     images: [
       {
-        url: `${deployUrl}/images/og.png`,
-        width: 184,
-        height: 78,
-        alt: 'cert-manager'
+        url: `${deployUrl}/images/og1.png`,
+        width: 1200,
+        height: 630,
+        alt: 'cert-manager - automated kubernetes X.509 certificates'
```

Comment:
Non-blocking: the deployUrl variable seems to not be correct on the `og:image` tag:

```html
<meta property="og:url" content="https://deploy-preview-963--cert-manager.netlify.app/">
<meta property="og:type" content="website">
<meta property="og:title" content="cert-manager">
<meta property="og:description" content="Cloud native X.509 certificate management for Kubernetes and OpenShift">
<meta property="og:image" content="https://cert-manager.io/images/og1.png">
```

Reply:
Yeah this is semi-intentional - we currently have the same og image for every page and because of _quirks_ it's much easier to hardcode the domain here. We lose the ability to preview the OG image but I didn't think it was worth walking the extra mile to fix the preview here. One for the future!

Reply:
I have opened https://github.com/cert-manager/website/pull/965 with a fix for that.